packages/ihe-gateway/server/ConfigurationMap.xml

```diff
@@ -1,93 +1,68 @@
 <?xml version="1.0" encoding="UTF-8"?><map>
   <entry>
-    <string>
-      SAML.SubjectRole.Code
-    </string>
+    <string>      IHE.XMLSchema</string>
     <com.mirth.connect.util.ConfigurationProperty>
-      <value>
-        224608005
-      </value>
```

Comment:
I know this is still draft, but flagging this so we don't remove stuff from the config map